Inspicio/webpack.config.js

```diff
@@ -12,3 +12,16 @@ module.exports = {
 		extensions: [".js"]
 	}
 };
+
+module.exports = {
+    entry: {
+        "ajaxScreens": './Scripts/ajaxScreens.js',
+    },
+    output: {
+        filename: "[name].bundle.js",
+        path: path.resolve(__dirname, 'wwwroot/dist/')
+    },
+    resolve: {
+        extensions: [".js"]
+    }
+};
```

Comment:
The exports block seems to have been accidentally duplicated here. It should be something like
```
 module.exports = {
 	entry: {
 		"images_view": './Scripts/images_view.js',
 		"ajaxScreens": './Scripts/ajaxScreens.js',	
 	},
 	output: {
 		filename: "[name].bundle.js",
 		path: path.resolve(__dirname, 'wwwroot/dist/')
 	},
 	resolve: {
  		extensions: [".js"]
  	}
  };
```

or just get rid of the `image_view.js` stuff (and the files that go with it)